### DISCUSSION TITLE
Facebook公司現在為Meta

### DISCUSSION THREAD

**Discussion by cobb110202008** [1101-GS3073-C/Python-Community-GS3073-C#94] [科技新知]
當下看到這消息，大大的震驚了我，Facebook竟然突然改名為Meta，一時之間令人難以接受，雖然Facebook創辦人馬克祖克伯提出了元宇宙（Meta verse）的概念，表示他們以後會以元宇宙為發展目標，元宇宙感覺就像是當今在線社交體驗的混合體，有時會擴展到三個維度或投射到物理世界，也能讓你與他人分享身臨其境的體驗 - 並一起做在現實世界中無法做的事情，但也引發外界的一些質疑與猜測，無論這個決定背後的原因為何，我都期待 Mata 公司能帶給我們更多的驚喜。
文章連結:https://www.bnext.com.tw/article/65884/facebook-rename-meta-2021

**Comment by pocper1** [comment]
老高的影片值得推薦XD
- [元宇宙](https://youtu.be/hm5K-PBz0rg)
- [NFT](https://youtu.be/cDk1FPoCfqI)

**Comment by qeeeee39** [comment]
不過有一說改名和轉換跑道是為了擋住近年越來越多的負面評論(特定詞語被祖...還有蒐集個資去做壞事的部分

**Comment by bonny123865** [comment]
突然改名感覺好奇怪！感覺會大眾跟媒體議論紛紛。
而以虛擬實境做為公司的未來願景，也讓我有點好奇會如何發展！

**Comment by briankuo123** [comment]
目前不太看好這個元宇宙。以現在的狀況來說，我認為他們做出來的東西只會被大眾認為是玩玩的而已。

**Comment by minniewei** [comment]
但我挺期待其作品的，感覺會給予我們不同的感受(畢竟fb已經長這樣太久了!!!)

**Comment by ayakasanma** [comment]
炒股無誤！目前技術還沒出來就已經吹成這樣....不單純

**Comment by Timmy110403540** [comment]
現在登入fb的時候，下面真的有個META字樣!

**Comment by wh20502** [comment]
我覺得現在社群軟體雖然更新很快，但IG人數的增加速度非常快，可能FB感受到威脅，趁這次熱潮趕快改名吧，比較有新鮮感

**Comment by Shiba010** [comment]
meta看起來是取metaverse元宇宙的前面，希望fb以後在這方面能有跨時代的發展。

**Comment by Lynn12378** [comment]
常常在小說和電影看過類似元宇宙的概念，希望FB能夠有重大的突破

**Comment by Yunhans** [comment]
Meta Knight
